### PR TITLE
Feature 1699: Added drop down to switch language for application (Internationalization)

### PR DESCRIPTION
**Pull request by variacode95**
Added drop down to select language (currently **English** and **Español**) in user profile.

Changing value in drop down will refresh the page and display content in respective language. 

We are passing a parameter `lang` with the URL, which will be stored in a cookie.

### CODE REVIEW

**Comment by miguelantonio** [comment]
@gschueler this is for the latin american spanish translation to work easly, also, should we squash the commits from now on as you indicated (so it doesn't show "Merge"?)

**Comment by gschueler** [comment]
@miguelantonio yes that would be best. It's cleanest to rebase your feature branch on the origin master, rather than merge in changes from it.

I updated our wiki with a typical git workflow at the bottom: https://github.com/rundeck/rundeck/wiki/Developer-Guidelines

File: rundeckapp/grails-app/views/user/profile.gsp [hunk 1]
```diff
@@ -101,6 +110,15 @@
             <g:message code="userController.page.profile.description" />
         </div>
     </div>
+    <div class="col-sm-12">
+        <span class="pull-right">
+            <label for="language">Language: </label>
```
Comment (miguelantonio) [inline rundeckapp/grails-app/views/user/profile.gsp]: Sure thing @variacode95 Parth could you add this please

**Comment by miguelantonio** [comment]
Thanks Greg, @variacode95 Parth for consideration, github does that as an option

File: rundeckapp/grails-app/views/user/profile.gsp (hunk 1 above)
Comment (variacode95) [inline rundeckapp/grails-app/views/user/profile.gsp]: Yes, @gschueler you're right. :) I'll make changes and will update the code.

**Comment by variacode95** [comment]
@miguelantonio @gschueler I've committed new changes that includes i18n properties for language, with re basing branch.

**Comment by gschueler** [comment]
why the merge commit?

**Comment by variacode95** [comment]
Messed up with the the merge commits, so created a new pull request https://github.com/rundeck/rundeck/pull/1919

**Pull request closed without merging**